lib/generators/shopify_app/shopify_app_generator.rb

```diff
@@ -2,8 +2,13 @@ module ShopifyApp
   module Generators
     class ShopifyAppGenerator < Rails::Generators::Base
 
+      def initialize(args, *options)
+        @opts = options.first
```

Comment:
Did you consider using `extract_options!`? https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/array/extract_options.rb#L22